web/src/beta/hooks/useManageSwitchState/hooks.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { renderHook, act } from "@testing-library/react";
+import { expect, test } from "vitest";
+
+import useManageSwitchState, { Props } from "./hooks";
+
+type SwitchField = {
+  id: string;
+  active: boolean;
```

Comment:
```suggestion
  name: string;
```
I think the props has `id` and `active` field in default, so could you set `name` to this field?
Also could you check if the name field is included in the fields state which is returned from the hooks?

Reply:
Added and committed.